views/layout.html

```diff
@@ -29,14 +29,24 @@
 {% block beforeMain %}
   {#  Headers or Navigation  #}
   <div class="absolute w-100 bg-white bg-transparent-ns z-1">
-    <nav class="w-site center w-80-ns">
+    <nav class="w-site center w-80-ns tr">
       <div class="center tc pt3 pl3">
         <a href="{{ data.home._url }}" class="fl">
-          <img src="../images/logo_na_dark.svg" alt="Quodl logo" class="w3-5">
+          {% if data.piece.tags == "" %}
+            <img src="../images/logo_na_dark.svg" alt="Quodl logo" class="w3-5">
+          {% else %}
+            <img src="../../images/logo_na_dark.svg" alt="Quodl logo" class="w3-5">
+          {% endif %}
```

Comment:
@Cleop I feel that there should probably be a nicer solution for this bit.
What is the difference in what should happen in the branches of this `if` statement? Do they both render the same thing?

Reply:
This is the only solution that I've been able to find. The two same things are rendered but basically Apostrophe will not load images correctly when you have a url with more than 1 `/` in the path which is what happens when you add a tag to a blog article, hence the if statement. 

The urls are built like this: `blog/tag(if the article is tagged)/name-of-article` - This doesn't work without the if

Whereas an article with no tag is simply: `blog/name-of-article` - This works

When you have more than one `\` in your path - `blog/students/name-of-article` then it will not find the image because it will be looking for it in the first part of the path ie. `blog/images/logo.svg` whereas it should simply be `images/logo.svg` so it needs an extra `../`. 

Ideally I'd make this change in the functionality when the tags are added but this is within the apostrophe node module.

What do you think @shouston3?

See #34 for my notes on how I came to this solution.

Reply:
I see, perhaps this is the best way (or at least a good way for now), could you just add a comment in the code just above the `if` statement pointing to #34.
(Just in case someone is reading through the code and wants to know why this was done)
Top marks for documenting your findings in #34 btw 👍